LoopKitUI/Views/Callout.swift

```diff
@@ -0,0 +1,120 @@
+//
+//  Callout.swift
+//  LoopKitUI
+//
+//  Created by Cameron Ingham on 1/5/24.
+//  Copyright © 2024 LoopKit Authors. All rights reserved.
+//
+
+import SwiftUI
+
+public struct Callout<Content: View>: View {
+    public enum Severity {
+        case note
+        case caution
+        case warning
+        
+        var hasIcon: Bool {
+            self != .note
+        }
+        
+        var prefix: Text {
+            switch self {
+            case .note:
+                return Text("Note: ")
+            case .caution:
+                return Text("Caution: ")
+            case .warning:
+                return Text("Warning: ")
+            }
+        }
+        
+        var color: Color {
+            switch self {
+            case .note:
+                return .accentColor
+            case .caution:
+                return .orange
```

Comment:
Assuming this is a change only for `Callout` and shouldn't persist to `GuidanceColors` warning (yellow) as well?

Reply:
Correct!